integration-tests/jest/jest-utils.js

```diff
@@ -0,0 +1,21 @@
+const { screenshotDir } = require('../utils/integration-config')
+
+
+const defaultTimeout = 5 * 60 * 1000
+
+const withGlobalJestPuppeteerContext = test => async () => await test({ context, page: await context.newPage() })
```

Comment:
nit: To avoid confusion with the global `test`, perhaps the argument should be named `fn`.